app/domain/authentication/authn_jwt/identity_providers/identity_from_url_provider.rb

```diff
@@ -0,0 +1,27 @@
+module Authentication
+  module AuthnJwt
+    module IdentityProviders
+      # Provides jwt identity from information in the URL
+      class IdentityFromUrlProvider < IdentityProviderInterface
+        def initialize(authentication_parameters)
+          @authentication_parameters = authentication_parameters
+        end
+
+        def jwt_identity
+          raise Errors::Authentication::AuthnJwt::IdentityMisconfigured unless identity_available
+
+          @authentication_parameters.username
+        end
+
+        def identity_available
```

Comment:
please see this: https://github.com/cyberark/conjur/pull/2227/files?file-filters%5B%5D=.feature&file-filters%5B%5D=.rb&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#r648215472

Reply:
It not opens me the link. Can you add screenshot?

Reply:
please verify that you perform this throughout the PR